### ISSUE TITLE
Render BBC news footer component

### ISSUE THREAD

**Issue by jamesbhobbs** [bbc/simorgh#55]
Blocked by: https://github.com/bbc/simorgh/issues/54

Create a new component in simorgh to render a very basic BBC News footer.

This should contain:
*A grey background
*Copyright text

Notes
*The text should be something along the lines of Copyright © 2018 BBC. The BBC is not responsible for the content of external sites. (TBC)
*We should try to automate this so we don't have to keep updating it
*We will build a component library to move this component to in a separate issue in the near future, but first we will do a basic render of our articles
